var/spack/repos/builtin/packages/keepassxc/package.py

```diff
@@ -65,6 +66,7 @@ def cmake_args(self):
             "-DCMAKE_INSTALL_DATADIR=%s" % join_path(spec.prefix, "share"),
         ]
         args.append(self.define_from_variant("WITH_XC_ALL", "autotype"))
+        args.append(self.define_from_variant("WITH_XC_AUTOTYPE", "autotype"))
```

Comment:
Can you say a bit more about the two options at lines 68-69? How do they differ?

Reply:
According to
https://github.com/keepassxreboot/keepassxc/wiki/Building-KeePassXC#build-options
XC_ALL enables all the plugin (or disable all) while XC_AUTOTYPE is just for a specific plugin.
Plugins require X11.
In spack I offer only all plugins or none (to make it simpler).

> -DWITH_XC_AUTOTYPE=[ON|OFF] Enable/Disable Auto-Type (default: ON)
> -DWITH_XC_YUBIKEY=[ON|OFF] Enable/Disable YubiKey HMAC-SHA1 authentication support (default: OFF)
> ... bla bla bla ...
> -DWITH_XC_KEESHARE=[ON|OFF] Enable/Disable KeeShare group synchronization extension (default: OFF)
> -DWITH_XC_ALL=[ON|OFF] Enable/Disable compiling all plugins above (default: OFF)
>